{{cookiecutter.repository_name}}/README.md

```diff
@@ -1,10 +1,116 @@
+
+# Install
+
+## Requirements
+
+**{{ cookiecutter.project_name }}** has been developed and tested on [Python {%- if cookiecutter.support_py2 == 'y' %}2.7, {%- endif %}3.4, 3.5, 3.6 and 3.7](https://www.python.org/downloads/)
+
+Also, although it is not strictly required, the usage of a [virtualenv](https://virtualenv.pypa.io/en/latest/)
+is highly recommended in order to avoid interfering with other software installed in the system
+where **{{ cookiecutter.project_name }}** is run.
+
+These are the minimum commands needed to create a virtualenv using python3.6 for **{{ cookiecutter.project_name }}**:
+
+```bash
+pip install virtualenv
+virtualenv -p $(which python3.6) {{ cookiecutter.repository_name }}-venv
+```
+
+Afterwards, you have to execute this command to have the virtualenv activated:
+
+```bash
+source {{ cookiecutter.repository_name }}-venv/bin/activate
+```
+
+Remember about executing it every time you start a new console to work on **{{ cookiecutter.project_name }}**!
+
+## Install with pip
+
+After creating the virtualenv and activating it, we recommend using
+[pip](https://pip.pypa.io/en/stable/) in order to install **{{ cookiecutter.project_name }}**:
+
+```bash
+pip install {{ cookiecutter.package_name }}
+```
+
+This will pull and install the latest stable release from [PyPi](https://pypi.org/).
+
+## Install from source
+
+Alternatively, with your virtualenv activated, you can clone the repository and install it from
+source by running `make install` on the `stable` branch:
+
+```bash
+git clone git@github.com:{{ cookiecutter.github_owner }}/{{ cookiecutter.repository_name }}.git
+cd {{ cookiecutter.repository_name }}
+git checkout stable
+make install
+```
+
+## Install for Development
+
+If you want to contribute to the project, a few more steps are required to make the project ready
+for development.
+
+First, please head to [the GitHub page of the project](https://github.com/{{ cookiecutter.github_owner }}/{{ cookiecutter.repository_name }})
+and make a fork of the project under you own username by clicking on the **fork** button on the
+upper right corner of the page.
+
+Afterwards, clone your fork and create a branch from master with a descriptive name that includes
+the number of the issue that you are going to work on:
+
+```bash
+git clone git@github.com:{your username}/{{ cookiecutter.repository_name }}.git
+cd {{ cookiecutter.repository_name }}
+git branch issue-xx-cool-new-feature master
+git checkout issue-xx-cool-new-feature
+```
+
+Finally, install the project with the following command, which will install some additional
+dependencies for code linting and testing.
+
+```bash
+make install-develop
+```
+
+Make sure to use them regularly while developing by running the commands `make lint` and
+`make test`.
+
+# Quickstart
+
+In this short tutorial we will guide you through a series of steps that will help you
+getting started with **{{ cookiecutter.project_name }}**.
+
+## 1. TODO: Put a title here
```

Comment:
In this case they do. This is the step number 1 of the step-by-step quickstart guide.
You can see an example of this here: https://github.com/DAI-Lab/TGAN#1-load-the-data

Reply:
Every subsection is a step and the steps obviously follow after each other. It's redundant to number the sections as well, including on the TGAN page

Reply:
I do not agree on the order thing. "Install from PyPI" and "Install from Source" are two subsections from the same section, and they have no order. The could be swapped and everything would be the same.

But, even if it could be argued that there is an implicit order, I do not see any harm on making it explicit in this case. Here there is a clear order, and we want to user to be completely aware of this order, so he starts running the things on step 1.

On the other side, this is the format that we have agreed on using on several other projects, not only the TGAN one, so I think this has to stay.

Reply:
My friend, it is obvious in a "Quickstart" or "Tutorial" section that steps are to be followed in order. It is simply poor style to number sections or subsections. By all means add numbers to sections or subsections on projects you maintain, but it is not necessary to impose that in this template.

Reply:
I updated the README removing the section altogether, and simply adding a `TODO` there indicating that the Quickstart needs to be written by the user.